docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_indexscrollbar.htm

```diff
@@ -54,8 +54,11 @@ <h2><a id='manual-constructor0.6990520032122731'></a>How to create index scroll
 <pre class="prettyprint">
```

Comment:
Global : Would you please check the existence of "\<p>HTML code:\</p>"?

Reply:
Fixed